README.md

```diff
@@ -20,7 +20,7 @@ The full list of changes for this release can be found in changelog.txt.
 
 ## System Requirements
 
-JDK 8 or above.
+JDK 17 or above.
```

Comment:
PR description says Java 11... ?

Reply:
Bump :) PR description is okay now, but not the backport, nor the Jira issue :see_no_evil: